README.md

```diff
@@ -1,7 +1,72 @@
+
+8. ```sudo apt-get update && sudo apt-get upgrade``` is also nice at this time to update potentially old packages.
+9. The image already comes with Jupyter Notebook, but it doesn't have the python2 kernel installed by default. This will cause issues if you run QNN tutorial notebooks on the board (as it is written in python 2), and if you plan to reuse some of the code in your own Jupyter Notebook. So we'll install the python2 ipython kernel.
+
+First we must upgrade pip, as the version included is quite old:
+```
+sudo pip install -U pip
+```
+
+Then we use pip to install the python2 kernel, and register the python 2 kernelspec. Credit: https://github.com/jupyter/jupyter/issues/71
+```
+sudo python2 -m pip install --upgrade ipykernel && sudo python2 -m ipykernel install
+```
+
+10. Next, we'd like for security reasons to not use default passwords, as it is always an awful practice not to change them. If on the QBART-team, there is a default password, ask a member. We need to update both sudo and user, so we run:
+```
+sudo passwd && passwd
+```
+
+We also need to change the password of the Jupyter server, reachable at <ip>:9090, as it is already running out of the box. From the PYNQ-docs:
+ hashed password is saved in the Jupyter Notebook configuration file.
+
+/root/.jupyter/jupyter_notebook_config.py
+You can create a hashed password using the function IPython.lib.passwd():
+
+```
+from IPython.lib import passwd
+password = passwd("secret") # Secret should be replaced by team password.
+6c2164fc2b22:ed55ecf07fc0f985ab46561483c0e888e8964ae6
+```
+
+You can then add or modify the line in the jupyter_notebook_config.py file
+
+```
+c.NotebookApp.password =u'sha1:6c2164fc2b22:ed55ecf07fc0f985ab46561483c0e888e8964ae6'
+```
+
+11. Lastly, we would most likely want to give each PYNQ a unique hostname, as the default "pynq" can be informative if one is to use several in a parallell OpenMPI approach. Use the provided PYNQ-script:
+```
+pynq_hostname.sh <NEW HOSTNAME>
+```
+Remember to reboot the board afterwards in order for changes to take effect.
+
+12. In addition, several other packages must be installed in order for the system to work:
+* 
+
+13. Finally, development files must be installed at the following locations:
+TODO: Add these when files becomem finished
 
-NB! Always remember to turn off the boards after use to reduce wear on the SD-card.
+13. It is also a pain in the butt to repeat this process for every single board, so in order to reduce the amount of work, one would ideally set up one board in the above manner, then we clone the SD-card to an image file, and then burn the image file to other PYNQ SD-cards. The image file should be provided on the repo for easy reuse. It is important to change hostname after flashing to other boards. Kinda pointless if all boards are named the same.
```

Comment:
Again, consider replacing "a pain in the butt" with something more professional like "strenuous" :sparkles: